src/ripple/app/tx/impl/Transactor.cpp

```diff
@@ -86,15 +86,30 @@ Transactor::Transactor(
 
 void Transactor::calculateFee ()
 {
-    mFeeDue = STAmount (getApp().getFeeTrack().scaleFeeLoad(
+    LoadFeeTrack& lft = getApp().getFeeTrack();
+
+    mFeeDue = STAmount (lft.scaleFeeLoad(
         calculateBaseFee(), view().fees().base,
-            view().fees().units, view().flags() & tapADMIN));
+            lft.getLoadBase(), view().flags() & tapADMIN));
 }
```

Comment:
I'm trying to not make them any more confusing than they already are.  But, yeah, I get confused.